texting/twilio.py

```diff
@@ -193,11 +194,19 @@ def send_sms(
                 return SendSmsResult(err_code=TWILIO_INVALID_TO_NUMBER_ERR)
         client = get_client()
         try:
-            msg = client.messages.create(
-                to=tendigit_to_e164(phone_number),
-                from_=tendigit_to_e164(settings.TWILIO_PHONE_NUMBER),
-                body=body,
-            )
+            if media_url is not None:
+                msg = client.messages.create(
+                    to=tendigit_to_e164(phone_number),
+                    from_=tendigit_to_e164(settings.TWILIO_PHONE_NUMBER),
+                    body=body,
+                    mediaUrl=media_url,
```

Comment:
It looks like the arg might be named `media_url`

Reply:
lol i just saw this comment after i made the fix. 🤞🏼 waiting for the linter now